test/cppsuite/create_test.sh

```diff
@@ -52,14 +52,14 @@ VALUE="#include \"$1.cxx\""
 sed -i "/$SEARCH/a $VALUE" $FILE
 
 # Add the new test to the run_test() method
-SEARCH="example_test(test_harness::test_args{config, test_name, wt_open_config}).run();"
```

Comment:
Fixed a few bugs in this script:

- We could not create a test that started with `t` because it would be translated into `\t` which is interpreted as a command.
- Tests end up being written on multiple lines, had to shorten what is being searched here